www/src/components/LayoutFooter/Footer.js

```diff
@@ -25,79 +25,132 @@ const Footer = () => (
     css={{
       backgroundColor: colors.darker,
       color: colors.white,
-      paddingTop: 50,
+      paddingTop: 10,
       paddingBottom: 50,
 
-      [media.size('xxlarge')]: {
+      [media.size('sidebarFixed')]: {
         paddingTop: 80,
       },
+
+      [media.size('sidebarFixedNarrowFooter')]: {
+        paddingTop: 10,
+      },
     }}>
-    <Container>
+    <Container isFooter={true}>
       <div
         css={{
           display: 'flex',
           flexDirection: 'row',
           flexWrap: 'wrap',
+
+          [media.between('small', 'medium')]: {
+            paddingRight: 240,
+          },
+
+          [media.between('large', 'xlargeSmaller')]: {
+            paddingRight: 280,
+          },
+          [media.between('xlargeSmaller', 'belowSidebarFixed')]: {
```

Comment:
I'm not sure I care for the newly-added media constants. The API was kind of built to make sense with _sizes_ but the new strings are kind of more arbitrary. This query particular is pretty confusing. 😁  What does it mean to be between _xlargerSmaller_ and _belowSidebarFixed_?